app/views/distributions/tumbleweed.html.erb

```diff
@@ -124,6 +125,38 @@
           <%= _('These ports are not officially supported, and they are not as stable as official distributions.') %>
         </div>
 
+        <h3 class="my-3">aarch64</h3>
```

Comment:
it is really horrible that we have to duplicate that much code when adding things... :confounded: we should find a better solution for that (partials for example)... but it was already wrong, so we can say that is unrelated to the PR :disappointed:

Reply:
Yes, I would say unrelated to this PR ;)